CHANGELOG.md

```diff
@@ -4,29 +4,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [0.1.0] - 2020-01-03
+## [Unreleased]
 
+## [0.1.0] - 2020-01-03
+### Added
 - Added ability to publish the container to DockerHub [#28](https://github.com/cyberark/conjur-api-python3/issues/28)
 - Changed test container to use Ubuntu instead of Alpine
 
-## [0.0.5]
+## [0.0.5] - 0000-00-00
 
 ### Added
-
 - Added ability to delete policies [#23](https://github.com/cyberark/conjur-api-python3/issues/23)
 
-## [0.0.4]
+## [0.0.4] - 0000-00-00
```

Comment:
Let's get the release dates in. Those are available from https://github.com/cyberark/conjur-api-python3/releases

Reply:
Fixed